packages/react-integration/demo-app-ts/src/components/demos/NumberInputDemo/NumberInputDemo.tsx

```diff
@@ -14,7 +14,7 @@ export class NumberInputDemo extends Component<NumberInputDemoState> {
 
   state: NumberInputDemoState = {
     value: 0,
-    value2: null
+    value2: undefined
```

Comment:
Maybe the test name should be changed at the end of numberInput.spec.ts as it is no longer testing null, and/or a null example added.

Might also be a good idea to cypress test the new functionality with:

- a backspace to create a value of "" and a blur to populate value
- typing beyond min/max and blur to see it snap to min/max